ONSdigital/eq-survey-runner-benchmark
# Include all endpoints in benchmark stats

### What is the context of this PR?
This PR includes all endpoints in our benchmark stats

### How to review 
- Ensure the stats include figures from all endpoints.
- Ensure graphing works as expected.
